src/renderer/library/components/Wizard.tsx

```diff
@@ -61,7 +64,7 @@ export const WizardModal = () => {
     const dispatch = useDispatch();
     const opened = useSelector((state: ILibraryRootState) => state.wizard.opened);
 
-    const [checked, setChecked] = React.useState(true);
+    const [checked, setChecked] = React.useState(false);
```

Comment:
I do not know the interaction between `state.wizard.opened` and the local `checked` state so I will let @panaC review this part.

Reply:
`state.wizard.opened` is the persisted opened state
`checked` is the local checkbox state mounted once

Reply:
sure, but how/when is `state.wizard.opened` mutated?
reading the code with my untrained eyes, I only see a read, not a write

Reply:
https://github.com/edrlab/thorium-reader/blob/2a9f7b2a962cb8ef80a2dcbc9d453144bdb8dec3/src/renderer/library/components/Wizard.tsx#L71